src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  const EROR_MESSAGE = 'Bzzz... Error!';
+  const maxCorectCountCall = 3;
+
+  return (x) => {
+    callCount++;
+
+    return (y) => callCount % 2 === 0 && callCount > maxCorectCountCall
```

Comment:
Create constant values for this `callCount % 2 === 0 && callCount > maxCorectCountCall`. 
Or even can split it into two variables